tests/integration/TEC/Tickets/Commerce/Admin_Tables/Orders_TableTest.php

```diff
@@ -4,54 +4,56 @@
 
 use TEC\Tickets\Commerce\Order;
 use tad\Codeception\SnapshotAssertions\SnapshotAssertions;
+use Tribe\Tests\Traits\With_Clock_Mock;
 use Tribe\Tickets\Test\Commerce\TicketsCommerce\Order_Maker;
 use Tribe\Tickets\Test\Commerce\TicketsCommerce\Ticket_Maker;
 use Tribe\Tests\Traits\With_Uopz;
 use TEC\Tickets\Commerce\Hooks;
 use Tribe\Tickets\Test\Traits\With_Globals;
 use WP_Screen;
 use WP_Query;
+use Tribe__Date_Utils as Dates;
 
 class Orders_TableTest extends \Codeception\TestCase\WPTestCase {
-
 	use SnapshotAssertions;
 	use Order_Maker;
 	use Ticket_Maker;
 	use With_Uopz;
 	use With_Globals;
```

Comment:
I added this trait `With_Globals` while developing this tests - you think its worth it to include it in common. Basically i was seeing my tests bleeding to other tests while i was editing global values

Reply:
Globals are reset by the test case at the end of the test ([link](https://github.com/lucatume/wp-browser/blob/v3.5/includes/core-phpunit/includes/abstract-testcase.php#L222) and [link](https://github.com/lucatume/wp-browser/blob/v3.5/includes/core-phpunit/includes/abstract-testcase.php#L187)), so a dedicated trait should not be required to perform that clean up.
What I ran into across were instances where overridden `setUp` or `tearDown` method were not invoking the parent correctly, leaking state.

Running the suite using plain global setting and relying on the test case to clean up got the job done; I would remove the trait.